satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -383,3 +383,28 @@ read one (
 	select certRecord
 	where  certRecord.id = ?
 )
+
+//--- satellite registration token for Vanguard release (temporary table) ---//
+
+model satellite_reg_token (
```

Comment:
`registration_tokens` - it's part of satellite dbx, so adding the extra prefix doesn't help much